src/radio-button/radio-button.styles.tsx

```diff
@@ -14,23 +15,32 @@ interface StyleProps {
 // =============================================================================
 // STYLING
 // =============================================================================
+
 export const Container = styled.div<StyleProps>`
     display: flex;
     justify-content: center;
     align-items: center;
-    height: 1.5rem;
-    width: 1.5rem;
+    height: 1.625rem;
+    width: 1.625rem;
```

Comment:
where does this value of `1.625rem` come from?

figma has a default size which is 2rem and a small size which is 1.5rem

Reply:
to switch to 2rem